config/the-config_top-section.md

```diff
@@ -117,7 +117,7 @@ Depending on the orientation of the board, you might need -20 degrees instead. (
 ```
 
 The values for the define are in degrees and in the order of `X,Y,Z`, the following image shows how these values translate to physical rotation of the board:
-<img src="https://pod.hubbe.net/images/board_orientation_rotation.svg" width=400 alt="orientation rotation">
+<img src="https://pod.hubbe.net/images/board_orientation_rotation.png" width=400 alt="orientation rotation">
```

Comment:
This image is much much better (assuming it's actually correct. :)

However, can we replace the two 'rotate-around-x' images with 'rotate-around-y' images instead?
The most common use case for this variable is hilts that have the board installed rotated around the Y angle.

Reply:
> (assuming it's actually correct. :)
So long as `monitor gyro` outputs as X,Y,Z and I'm not blind it is :laughing: 

> However, can we replace the two 'rotate-around-x' images with 'rotate-around-y' images instead?
I'd just need a side view image of the board... I couldn't find one and I don't have a great camera/setup to take one (also, not chomping at the bit to disassemble a chassis to extract the board)

Would agree Y images would be more common/more useful.

Reply:
> I'd just need a side view image of the board

There's this:
https://github.com/profezzorn/ProffieOSDocs/blob/main/images/board_orientations.png

Reply:
@ryryog25 @profezzorn 
What would be the coolest is a 3D board you can drag around with your mouse and show real time X, Y, Z values as you rotate it around. Maybe include axis lock check boxes to isolate the one(s) you want. Anyone know how to do that? I'm thinking the Shapeways 3D preview when viewing chassis and stuff.

Reply:
> There's this: https://github.com/profezzorn/ProffieOSDocs/blob/main/images/board_orientations.png

Yes, I considered using that, but was hesitant since it ends up being only a handful of pixels wide once the board view is extracted.

Reply:
> What would be the coolest is a 3D board you can drag around with your mouse and show real time X, Y, Z values as you rotate it around. 

Web dev certainly isn't my thing, nor is something of that scale for, as Fredrik pointed out, a lesser-used feature something I'd find it worth my time to do it if I was familiar enough to do without the time investment to learn it all.

Reply:
I can certainly commit this as is and then we can improve it later.
I do want to make sure that the angles have actually been verified though.
I can test it myself tomorrow, unless it's already been tested?

Reply:
I’ve connected the board via serial monitor and used `monitor gyro` to verify those directions. If you or @NoSloppy want/are able to double check them I’d certainly welcome it. 

Unless I misunderstand how the values work (very possible), when rotating the board in the directions given by the arrows, gyro readings spike positively in the respective axis, that’s what the current visual description is based off of.

Reply:
That's true, but I also wanted to make sure that ORIENTATION_ROTATION works as intended.
That means turning the board 30 degrees in some direction, update ORIENTATION_ROTATION to compensate, and then verify that the "monitor gyro" and "monitor swings" behave as if the board was not rotated.
I have a sneaky suspicion that the sign used in the ORIENTATION_ROTATION might be backwards...

Reply:
Yeah, that kind of testing I have not done, and that idea of the direction being reversed when using ORIENTATION_ROTATION as though it’s a measure of compensation seems fairly possible.

Reply:
@profezzorn I'm just now testing this, and when setting `ORIENTATION_ROTATION 30,0,0` and rotating the board like shown in the image (I felt this one was the easiest for me to limit rotation in other directions), while I hold it unrotated and twist it about the Z axis, both Y and Z spike, and when rotated as shown in the image (30 degrees clockwise), there are only spikes for Z.

Based off of that axis it seems that these values actually aren't backwards...

Reply:
bump